src/_cffi_src/openssl/ssl.py

```diff
@@ -520,6 +523,7 @@
 static const long Cryptography_HAS_SSL_OP_NO_TICKET = 1;
 static const long Cryptography_HAS_SSL_SET_SSL_CTX = 1;
 static const long Cryptography_HAS_NEXTPROTONEG = 1;
+static const long Cryptography_HAS_KEYING_EXPORT = 1;
```

Comment:
This (and the declaration) are also unnecessary now. Other than that this looks good!

Reply:
you beat me to it :)